src/components/Dashboard.scss

```diff
@@ -93,3 +95,79 @@
     }
   }
 }
+
+.main-layout__header {
+  .main-header__logo {
+    span {
+      font-size: 0.9rem;
+    }
+  }
+}
+
+@media screen and (max-width: 1201px) {
+  .converter {
+    display: flex;
+    flex-direction: column-reverse;
+    align-items: center;
+
+    .converter__info {
+      &__content {
+        &::before {
+          display: none
+        }
+      }
+    }
+
+    .converter__calc {
+      width: 500px;
+    }
+  }
+}
+
+@media screen and (max-width: 576px) {
+
+  .main-layout__header {
+    display: flex;
+    align-items: center;
+
+    .main-header {
+      &__logo {
+        font-size: 1.5rem;
+
+        span {
+          font-size: 0.6rem;
+        }
+      }
+
+      .ant-menu {
+        padding: 0 10px;
+
+        .ant-menu-item {
+          padding: 0 10px;
+
+          span {
+            display: none;
+          }
+        }
+      }
+    }
+  }
+
+  .main-layout__content {
+    padding: 0 10px;
+
+    .converter__info__content {
+      font-size: 1rem;
```

Comment:
`1rem` is default one. Are there any visual changes with this style definition? 🙈

Reply:
Yes, there is visual change ;)